vendor/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -32,7 +32,6 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/golang/glog"
-	"k8s.io/kubernetes/federation/apis/federation"
```

Comment:
@rhcarvalho This was most likely taken out after rebase with new version of kube